bindings/nodejs/generated.d.ts

```diff
@@ -28,6 +28,29 @@ export class ExternalObject<T> {
     [K: symbol]: T
   }
 }
+/** Metadata carries all metadata associated with a path. */
+export interface Metadata {
```

Comment:
This change seems like a breaking change, how about seperating into a new PR so we can track it better in our changelogs?